src/main/kotlin/no/nav/syfo/service/SykmeldingService.kt

```diff
@@ -176,21 +181,25 @@ class SykmeldingService constructor(
                     PAPIRSM_MAPPET.labels("feil").inc()
                     log.warn("Noe gikk galt ved mapping fra OCR til sykmeldingsformat: ${e.message}, {}", fields(loggingMeta))
 
-                    val sakId = sakClient.finnEllerOpprettSak(sykmeldingsId = sykmeldingId, aktorId = aktorId, loggingMeta = loggingMeta)
-
-                    val oppgave = oppgaveService.opprettOppgave(aktoerIdPasient = aktorId, sakId = sakId,
-                            journalpostId = journalpostId, gjelderUtland = false, trackingId = sykmeldingId, loggingMeta = loggingMeta)
-
-                    if (!oppgave.duplikat) {
-                        log.info("Opprettet oppgave med {}, {} {}",
-                                StructuredArguments.keyValue("oppgaveId", oppgave.oppgaveId),
-                                StructuredArguments.keyValue("sakid", sakId),
-                                fields(loggingMeta)
-                        )
-                        PAPIRSM_OPPGAVE.inc()
+                    val papirSmRegistering = mapOcrFilTilPapirSmRegistrering(
```

Comment:
Det er vel i denne delen av koden at vi kun skal gjøre dette hvis brukeren tilhører et av pilotkontorene..? Og hvis bruker ikke tilhører pilotkontor så skal vi gjøre sånn som vi gjorde før? Siden det innebærer en sjekk mot arbeidsfordeling og generelt kan bli litt...mye... bør vi kanskje trekke ut hele feilhåndteringen i en egen service. Vi kan gruble litt på hva som blir det ryddigste her :)

Reply:
Ja, jeg er ikke helt sikker på hvordan det bør gjøres best.

Det var bra dette kom opp på dagens statusmøte, for jeg var ikke helt klar over implikasjonene av dette utover at jeg registrerte at Joakim skrev "OBS ikkje merge, før vi har funnet ut hvem som er pilot kontor".